rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -44,12 +48,13 @@ def required_packages(cls):
         return ["duckling"]
 
     @classmethod
-    def create(cls, duckling_processing_mode):
-        if duckling_processing_mode not in DUCKLING_PROCESSING_MODES:
-            raise ValueError("Invalid duckling processing mode. Got '{}'. Allowed: {}".format(
-                duckling_processing_mode, ", ".join(DUCKLING_PROCESSING_MODES)))
+    def create(cls, duckling_dimensions=DUCKLING_DIMENSIONS):
```

Comment:
shouldn't take default arguments as these are filled using reflection and the default will always be overwritten